Box.V2/Managers/BoxCollaborationsManager.cs

```diff
@@ -51,11 +51,16 @@ public BoxCollaborationsManager(IBoxConfig config, IBoxService service, IBoxConv
         }
 
         /// <summary>
-        /// Used to edit an existing collaboration. Descriptions of the various roles can be found 
-        /// <see cref="https://support.box.com/entries/20366031-what-are-the-different-collaboration-permissions-and-what-access-do-they-provide"/>
+        ///Used to edit an existing collaboration. Descriptions of the various roles can be found here
         /// </summary>
-        /// <param name="collaborationRequest"></param>
-        /// <returns></returns>
+        /// <param name="collaborationRequest">
+        /// collaborationRequest.Id (Required) - Id of the collaboration object
+        /// collaborationRequest.Role (Required) - The access level of this collaboration
+        /// collaborationRequest.Status - Whether this collaboration has been accepted
+        /// </param>
```

Comment:
I think you should put a comment for fields from https://docs.box.com/reference#fields